recipes/proxsuite/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+if [ "$(uname)" == "Darwin" ]; then
+  # for Mac OSX
+  export CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
+fi
+
+mkdir build
+cd build
+
+cmake ${CMAKE_ARGS} .. \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DBUILD_TESTING:BOOL=OFF \
+      -DINSTALL_DOCUMENTATION:BOOL=ON \
+      -DBUILD_PYTHON_INTERFACE:BOOL=ON \
+      -DBUILD_WITH_VECTORIZATION_SUPPORT:BOOL=ON \
+      -DPYTHON_EXECUTABLE=$PYTHON
+
+make -j${CPU_COUNT}
+make install
```

Comment:
Is there a `make check` or `ctest` we can run here?

Reply:
It will take a while to also account for all the tests. We just check for the correct import of the Python lib. Is it fine?

Reply:
@ocefpaf Is it fine for you?